ddl/stat_test.go

```diff
@@ -89,7 +89,7 @@ func TestDDLStatsInfo(t *testing.T) {
 			varMap, err := d.Stats(nil)
 			wg.Done()
 			require.NoError(t, err)
-			require.Equal(t, "1", varMap[ddlJobReorgHandle])
+			require.Equal(t, "", varMap[ddlJobReorgHandle])
```

Comment:
why the test is changed

Reply:
Because the channel `ddl.TestCheckWorkerNumCh` is send, the transaction that updating `ddl_job_reorg_handle` is not committed.